src/uct/tcp/sockcm/sockcm_ep.c

```diff
@@ -64,7 +65,10 @@ ucs_status_t uct_sockcm_ep_send_client_info(uct_sockcm_ep_t *ep)
         goto out;
     }
 
-    conn_param.length = ep->pack_cb(ep->pack_cb_arg, dev_name,
+    pack_args.field_mask = UCT_CM_EP_PRIV_DATA_PACK_ARGS_FIELD_DEVICE_NAME;
+    ucs_strncpy_safe(pack_args.dev_name, dev_name, UCT_DEVICE_NAME_MAX);
+
+    conn_param.length = ep->pack_cb(ep->pack_cb_arg, &pack_args,
```

Comment:
maybe add wrapper: uct_cm_client_pack_cb() like for others?

Reply:
@yosefe you mean a non API function right? in this PR then?

Reply:
yes, non-API, in this PR